package com.caredear;

message NetdiskRequest{
    required string user = 1;  // user's mobile number(such as 130xxxxxxxx)
    required string md5 = 2;
    required string filename = 3;
    required int32  filesize = 4;
}

message NetdiskResponse{
    required int32 result_code = 1; //CDS_XXX error code
    required string uploadtoken = 2;
    required string downloadurl = 3;
    required string netdisckey = 4;
}
